acmw-db-app/components/AddAccountForm.jsx

```diff
@@ -54,21 +57,33 @@ const AddAccountForm = (props) => {
           const requestOptionsStudent = {
             method: 'POST',
             body: JSON.stringify(
-              { fname: fname.charAt(0).toUpperCase() + fname.slice(1).toLowerCase(), //capitalization convention
-                lname: lname.charAt(0).toUpperCase() + lname.slice(1).toLowerCase(), //capitalization convention
+              { fname: fname,
+                lname: lname,
                 name_dot_num: lnamedotnum.toLowerCase(),
                 personal_email: "", //these are blank for now
                 school_level: "",
                 packet_sent_date: ""
               }
             )
           };
-          const res1 = await fetch('/api/student/create', requestOptionsStudent);
-          const result1 = await res1.json(); //returns the id
-          id = result1[0]["id"];
+          try {
+            const res1 = await fetch('/api/student/create', requestOptionsStudent);
+            const result1 = await res1.json(); //returns the id
+            id = result1[0]["id"];
+          } catch (err) {
+            // hopefully duplicate; selects student instead
```

Comment:
this should use the error code that corresponds to attempted duplication, not another query